bundles/org.openhab.binding.teleinfo/src/main/resources/OH-INF/thing/common-lsm-channel-group.xml

```diff
@@ -0,0 +1,362 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="teleinfo"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="https://openhab.org/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/thing-description/v1.0.0 https://openhab.org/schemas/thing-description-1.0.0.xsd">
+
+	<channel-group-type id="commonLSMGroupType">
+		<label>CommonLSMChannelGroup</label>
+		<description>Linky Standard Mode common channels</description>
+		<category>Energy</category>
+		<channels>
+
+			<channel id="ngtf" typeId="stringType">
+				<label>NGTF</label>
```

Comment:
Does the normal user know what all these abbreviations stand for?

Reply:
Honestly I don't know!  Here are my points for not changing this:
* Non expert users may use the linky binding which requires only an internet connection. Teleinfo requires buying a modem and installing wires between the telemeter and the server so I think it implies some advanced knowledge on the teleinfo protocol capabilities.
* The old channels already have this kind of label.
* These labels follow strictly the teleinfo protocol specification (see https://www.enedis.fr/sites/default/files/Enedis-NOI-CPT_54E.pdf, first link of the readme).
* I've added more than 90 channels in this PR so It will be a very huge work to find a best suited label for each new channel. And I'm not sure to understand perfectly the meaning of each channel so I may introduce some defaults.

Reply:
Your points sound resonable.